docker-compose.yml

```diff
@@ -235,4 +235,4 @@ services:
     environment:
       - JAVA_OPTS=-Dwebdriver.chrome.whitelistedIps=
       - VIRTUAL_PORT=7900
-      - VIRTUAL_HOST=chrome.ams.test
+      - VIRTUAL_HOST=chrome.hyku.test
```

Comment:
NOTE: I don't understand why this fixes it. And I also don't understand why we can connect to hyku.test when it's not defined in our docker setup. 🤔 No other docker projects are running so this must be coming from gbh's setup somehow. I think it's ok to proceed for now since it unblocks us however Rob should prob look into it.